cmd/mockery.go

```diff
@@ -75,6 +75,7 @@ func NewRootCmd() *cobra.Command {
 	pFlags.Bool("disable-version-string", false, "Do not insert the version string into the generated mock file.")
 	pFlags.String("boilerplate-file", "", "File to read a boilerplate text from. Text should be a go block comment, i.e. /* ... */")
 	pFlags.Bool("unroll-variadic", true, "For functions with variadic arguments, do not unroll the arguments into the underlying testify call. Instead, pass variadic slice as-is.")
+	pFlags.Bool("omit-empty-rolled-variadic", false, "For functions with variadic arguments, omit passing empty slice to underlying testify call. Used in tandem with unroll-variadic set to false.")
```

Comment:
Not super happy with the naming here.
Ideally `unroll-variadic` would be called `roll-variadic` and defaulted to false. We could then call this new option `roll-variadic-omit-empty` which feels better.
Curious - how do we feel about deprecating `unroll-variadic` in place of `roll-variadic`?

Reply:
Deprecating the variable name is probably a lot of effort. Technically speaking, "unroll" makes more sense as it is "expanding"/"unrolling" the slice into the variadic `mock.On(` testify call.

Reply:
Sure but the inverse of unrolling is rolling and you avoid double negatives. Agreed that deprecating wouldn't be easy but we should rather question whether it's correct.

Reply:
A part of me also feels like, if we do add this feature, it doesn’t need to be behind a feature flag. I can’t think of a case where this would break someone relying on the current logic?